public/tutorials/cross-dom-solidity.js

```
@@ -1,37 +1,47 @@
-  l1ChainId: 11155111, // 11155111 for Sepolia, 1 for Ethereum
-  l2ChainId: 11155420, // 11155420 for OP Sepolia, 10 for OP Mainnet
-  l1SignerOrProvider: l1Wallet,
-  l2SignerOrProvider: l2Wallet,
-})
+const l1Provider = createPublicClient({ chain: sepolia, transport: http("https://rpc.ankr.com/eth_sepolia") }).extend(publicActionsL1())
+const l2Provider = createPublicClient({ chain: optimismSepolia, transport: http("https://sepolia.optimism.io") }).extend(publicActionsL2());
 
 console.log('Waiting for message to be provable...')
-await messenger.waitForMessageStatus(transactionHash, optimism.MessageStatus.READY_TO_PROVE)
+await l1Provider.getWithdrawalStatus({ 
+  receipt, 
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)